README.md

```diff
@@ -78,6 +70,10 @@ In case this error occurs, the subsystems list in testutils.py should be updated
 
```

Comment:
This doesn't include enums and tables directories, which hold constants. Can we include that?

## Constanst

Apart from Python code to manipulate XML definition, the repository contains code with enumeration values and tables used by projects. Those are stored in (https://github.com/lsst-ts/ts_xml/tree/main/python/lsst/ts/xml/enums) and (https://github.com/lsst-ts/ts_xml/tree/main/python/lsst/ts/xml/tables).